travvy/src/components/Home.vue

```diff
@@ -5,7 +5,7 @@
   <div class="Home">
     <div class="container">
     <!-- Profile button that will lead to the users profile page -->
-    <input type="button" value="Profile" v-on:click="reverseMessage" style="float: right;" class="profile">
+    <router-link to="/Profile" tag="button" class="button" style="float: right;"><span>Profile</span></router-link>
```

Comment:
We've had a few problems with router-link, might be a better idea to use a button that takes you to profile

Reply:
that was my original plan but I saw the use of router-link for some other buttons and thats why I used it